.gitignore

```diff
@@ -1,5 +1,6 @@
 # CMake build and local install directory
 _skbuild
+build
```

Comment:
Curiosity: why do you find this necessary?

Reply:
It's a folder generated during the build, isn't it?

Reply:
It used to, when we used `setuptools`. With transition to `scikit-build` the generated folder is `_skbuild`.

Reply:
Looks I've not cleaned up the env for a long time. I will roll back the change.